php/classes/handlers/class-ajax-handler.php

```
@@ -345,6 +349,69 @@ public function import_external_rss_feed() {
+	 * Sends a feed summary, or the reason the feed cannot be imported, as JSON.
+	 *
+	 * @since 3.18.0
+	 *
+	 * @param array|\WP_Error $result Result from the onboarding import handler.
+	 */
+	protected function send_feed_result( $result ) {
+		if ( is_wp_error( $result ) ) {
+			wp_send_json(
+				array(
+					'status'  => 'error',
+					'code'    => $result->get_error_code(),
```

PHPStan (2.2.8)

[error] Cannot call method get_error_code() on array|WP_Error.


(method.nonObject)